src/plugins/orangehrmAdminPlugin/test/Dao/ModuleDaoTest.php

```diff
@@ -69,7 +69,7 @@ public function testUpdateModuleStatusForReturnedValues(): void
     {
         $moduleUpdateArray = ['admin' => true, 'pim' => false];
         $returedObjects = $this->moduleDao->updateModuleStatus($moduleUpdateArray);
```

Comment:
seems like having spelling mistake in previous code. better to fix 
```suggestion
       $returnedObjects = $this->moduleDao->updateModuleStatus($moduleUpdateArray);
```